apps/web-connect/src/features/checkout/components/ActionSection.tsx

```diff
@@ -64,23 +65,23 @@ export function ActionSection(): JSX.Element {
                 {currency === 'AETH' ? (
                     <PrimaryButton
                         onClick={() => mintWithEth.write?.()}
-                        className={`w-full h-16 ${ready && chain?.id === 42161 ? "bg-[#F30919] global-clip-path" : "bg-gray-400 cursor-default !text-[#726F6F]"} text-lg text-white p-2 uppercase font-bold`}
-                        isDisabled={!ready || chain?.id !== 42161}
+                        className={`w-full h-16 ${ready ? "bg-[#F30919] global-clip-path" : "bg-gray-400 cursor-default !text-[#726F6F]"} text-lg text-white p-2 uppercase font-bold`}
+                        isDisabled={!ready}
                         btnText={getButtonText()}
                     />
                 ) : (
                     <PrimaryButton
                         onClick={handleBuyWithXaiClicked}
-                        className={`w-full h-16 ${ready && chain?.id === 42161 ? "bg-[#F30919] global-clip-path" : "bg-gray-400 cursor-default !text-[#726F6F]"} text-lg text-white p-2 uppercase font-bold`}
-                        isDisabled={!ready || chain?.id !== 42161 || !userHasTokenBalance}
+                        className={`w-full h-16 ${ready ? "bg-[#F30919] global-clip-path" : "bg-gray-400 cursor-default !text-[#726F6F]"} text-lg text-white p-2 uppercase font-bold`}
+                        isDisabled={!ready || !userHasTokenBalance}
```

Comment:
Same as the comment above.

We only would remove these checks on a sepolia testnet build.

Reply:
I believe these are all cleaned up now. Thank you for the guidance!